test/plots/grid-choropleth.js

```diff
@@ -0,0 +1,38 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const [grid, data] = await Promise.all([
+    await d3.csv("data/us-state-grid.csv", d3.autoType).then(gridmap),
+    await d3.csv("data/us-state-population-2010-2019.csv", d3.autoType)
+  ]);
+  const states = data
+    .filter(d => grid.has(d.State))
+    .map(d => ({...d, ...grid.get(d.State)}));
+  return Plot.plot({
+    height: 420,
+    x: {
+      axis: null
+    },
+    y: {
+      axis: null
+    },
+    color: {
+      type: "diverging",
+      scheme: "piyg"
+    },
+    marks: [
+      Plot.cell(states, {x: "x", y: "y", fill: change}),
```

Comment:
```suggestion
    color: {
      type: "diverging-log",
      scheme: "piyg"
    },
    marks: [
      Plot.cell(states, {x: "x", y: "y", fill: d => 1 + change(d)}),
```

percentage changes should be log1p-scaled; it's pedantic here, since there is no noticeable difference with the values in this plot. But if you multiply line 37 by 30, the lowest value becomes -99%, undoubtedly the strongest, and it should determine the scale more than the value for DC (which has then become a mere +519%). Besides being a "best practice", this also gives us an opportunity to have a unit test for diverging-log color scales.